### DISCUSSION TITLE
Two instances of Music Assistant? Cannot get one of them to start up on Home and not Settings

### DISCUSSION THREAD

**Discussion by asinla** [music-assistant/support#2340] [Q&A]
I'm running HA ina VMWARE VM on a Macbook Pro. Works really well.

I had been running MA successfully since the first public beta. 

Since the official release it now seems I had two MA integrations running. One was Music Assistant and one was still Music Assistant Beta. I could not get either of those to open on the Home page of MA. It would just go to Settings.

Today I completely removed all integrations and Add-Ons to reinstall from scratch. I followed the instructions to install the HA Add On as I am running HAOS.
It installed but all my previous settings were still there. My services, players and settings. And now when I restart HA auto discovers a Music Assistant Beta that it wants me to configure. And I STILL cannot get it to open on the Home page and not Settings!

I'm sure I'm doing something wrong but cannot figure out what it is. 

Home Assistant Core - 2024.5.4
Supervisor - 2024.05.1
OS - 12.3

Music Assistant - Server Version 2.0.4
Music Assistant HACS Integration - 2024.5.1

**Comment by OzGav** [comment]
You must still have two versions of the addon running. Look in HA SETTINGS>>ADDONS

**Reply by asinla** [reply]
Thanks. I did check that. Only one there as far as I can tell. Here is a screen shot.

![Screenshot 2024-05-20 at 3 47 05 PM](https://github.com/music-assistant/hass-music-assistant/assets/112218272/1f4ffc2e-75a0-469c-8daf-c223b9942894)

And here is a screen shot of the Integrations page. As you can see it's discovering a BETA version there.


![Screenshot 2024-05-20 at 3 49 09 PM](https://github.com/music-assistant/hass-music-assistant/assets/112218272/5a757448-358f-4bce-a6f6-02a4d2e94b7d)

**Reply by OzGav** [reply]
No that is the HA integration not the ADDON so that is normal. The integration will have the beta label removed at the next update

**Reply by asinla** [reply]
So when I click on the ADDON and click Open Web UI it takes me to the Music Assistant Settings page. I can see my Music Providers, Player Providers and Metadata Providers. From there I seem to be able to use MA as usual.

Are you saying I should choose to configure the Integration that has been discovered and not use the Web UI from the ADDON?

**Reply by OzGav** [reply]
No they are two different things https://music-assistant.io/#architecture

The settings page opening is an issue to be fixed

**Reply by asinla** [reply]
Ok thanks.

So just to be clear, I should click "Configure" on my HA INTEGRATIONS page and go from there?

**Reply by OzGav** [reply]
Yes if you want to use the Integration. It is not mandatory.

**Reply by asinla** [reply]
What are the differences between using the Integration or just the Webui?

**Reply by OzGav** [reply]
I don’t understand the question. The link I provided above describes what these two components do.

**Reply by asinla** [reply]
Thanks.
